src/schema/v2/payment_device_union.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  GraphQLBoolean,
+  GraphQLObjectType,
+  GraphQLUnionType,
+  GraphQLNonNull,
+} from "graphql"
+import { BankAccountType } from "schema/v2/bank_account"
+import { CreditCardType } from "schema/v2/credit_card"
+
+export const ManualPaymentType = new GraphQLObjectType({
+  name: "ManualPayment",
+  fields: () => ({
+    isManualPayment: {
+      type: new GraphQLNonNull(GraphQLBoolean),
+      resolve: () => true,
```

Comment:
Do we need to have this field at all? I don't know if objects that only have a `__typename` are allowed (I seem to recall they weren't in the past but i may be confusing things). 

Alternately it could be `isWireTransfer: true` to allow for other types of manual payment in the future - thus giving us a _reason_ to have the field.